src/components/EditorNav.tsx

```diff
@@ -169,7 +174,7 @@ export default function EditorNav({
             size="sm"
             className={"h-6 px-2 text-xs save-button"}
             disabled={!isLoggedIn || !canSave}
-            onClick={onSave}
+            onClick={handleSave}
```

Comment:
Unnecessary change, more complex and doesnt do anything because the button is disabled

Reply:
Done : )